scripts/example.jl

```diff
@@ -2,26 +2,27 @@
 using FreeBird
 
 # First, we need to read the walkers from a file
+# Note that the periodic boundary conditions are updated to True, True, False
 ats = read_walkers("scripts/slab-08-4x4.extxyz",pbc="TTF")
-# Note that the periodic boundary conditions are set to True, True, False
+
+# Update the system to have 80 frozen particles and 8 free particles
+ats = [AtomWalker{2}(at.configuration;list_num_par=[80,8],frozen=Bool[1,0]) for at in ats]
 
 # Then, we need to create a potential object
-lj = LJParameters(epsilon=0.1, sigma=2.5, cutoff=4.0, shift=0.5)
+lj = LJParameters(epsilon=0.1, sigma=2.5, cutoff=4.0, shift=true)
 
 # Now, we can create the liveset by combining the walkers and the potential
-ls = LJAtomWalkers(ats[1:10], lj, 80)
-# The last argument is the number of frozen particles
+ls = LJAtomWalkers(ats, lj)
 
 # We can now create a Monte Carlo object
-mc = MCDemonWalk(max_add_steps=10000, e_demon_tolerance=0.01u"eV")
-mix = MixedMCRoutine(main_routine=MCRandomWalk(),back_up_routine=mc)
+mc = MCRandomWalkClone()
 
-# We can also create a save object
-save = SaveEveryN("output-energies.csv",100)
+ns_params = NestedSamplingParameters(1600, 0.1, 0.01, 1e-5, 0.1, 0, 200)
+# We can also create a save object, using default value for most parameters except n_snap
+save = SaveEveryN(n_snap=1000)
 
 # And finally, we can run the simulation
-energies, ls, _ = nested_sampling_loop!(ls, 10000, mix, save)
+energies, ls, _ = nested_sampling_loop!(ls, ns_params, 10_000, mc, save)
```

Comment:
Is 10,000 the number of nested sampling iterations?

Reply:
Yes. The number of decorrelation steps is defined within `ns_params`.